### ISSUE TITLE
check for valid hayagriva

### ISSUE THREAD

**Issue by wuyoli** [typst/hayagriva#134]
i would like to automatically check if a file is valid hayagriva. (preferably with helpful error messages if not.) Is this currently supported or planned?

**Comment by DerDrodt** [comment]
You are asking for a `check` command or something along those lines? As far as I now, there is no work on that at the moment. But it is a good idea.

**Comment by wuyoli** [comment]
yes, a `check` command would be pretty much exactly what I was thinking off. From what i understand this should be relatively easy? (just return error code when parsing was unsuccessful?) the error messages would likely be more work though.
